packages/docs/src/components/content/PropTableHtmlElementRow.tsx

```diff
@@ -0,0 +1,37 @@
+import humanizeList from 'humanize-react';
+import { TableRow, TableCell } from '@cmsgov/design-system';
+
+export interface PropTableHtmlElementRowProps {
+  /**
+   * List of element names
+   */
+  elements: string[];
+}
+
+/**
+ * If the component we're documenting passes extra props through to an HTML element,
+ * we want to include documentation about that. Rather than listing out all the
+ * additional attributes for those HTML elements in our props table, we will add
+ * some explanatory text with MDN documentation links to the bottom of our props
+ * table.
+ */
+const PropsTableHtmlElementRow = ({ elements }: PropTableHtmlElementRowProps) => {
+  const elementLinks = elements.map((name) => (
+    <a href={`https://developer.mozilla.org/en-US/docs/Web/HTML/Element/${name}`} key={name}>
+      {`<${name}>`}
+    </a>
+  ));
+  const formattedElementLinks = humanizeList(elementLinks, { conjunction: 'or' });
+
+  return (
+    <TableRow>
+      <TableCell colSpan={4}>
+        This component passes any additional props to its underlying {formattedElementLinks} element
+        as attributes. It will accept any props that are valid attributes of {formattedElementLinks}
```

Comment:
```suggestion
        as attributes. See MDN for a list of valid attributes for {formattedElementLinks}.
```

Unlink the code block and show as inline and link up MDN text instead. I think is more understandable about where the link will be taking the user.

Reply:
I had it like that in my first iteration, but 2 of the 6 components have a choice between two different underlying elements. How would I link to each one in that case and make it clear which link is which? That's why this version uses an array for the element(s) prop.

Reply:
Ah gotcha, maybe then adding some text about the MDN would be helpful. Right now it's unclear where the link is going or if it's external

Reply:
Here's what it looks like now: 

<img width="891" alt="Screen Shot 2022-11-07 at 4 11 40 PM" src="https://user-images.githubusercontent.com/7595652/200442527-3a1b7ed3-c132-4e08-9ffb-20c922127b1b.png">